### ISSUE TITLE
CC doesn't appear on macOS when VO is running

### ISSUE THREAD

**Issue by julianna-langston** [julianna-langston/chart2music#571]
1. On a MacOS device, turn on VoiceOver, and open chart2music.com
2. Navigate to the chart
3. Press left/right arrow keys.

**Current behavior**
Sounds play, but no text is read out. This is because the CC element isn't visible.

**Correct behavior**
The text should be read out by VoiceOver.

Note - If VoiceOver isn't running, the CC element appears.
